src/SourceBuild/content/build.sh

```diff
@@ -119,10 +119,10 @@ while [[ $# > 0 ]]; do
     # Source-only settings
     -source-only|-so)
       sourceOnly=true
-      properties="$properties /p:DotNetBuildFromSource=true"
+      properties="$properties /p:DotNetBuildFromSource=true /p:DotNetBuildSourceOnly=true"
       ;;
     -online)
-      properties="$properties /p:BuildWithOnlineSources=true"
+      properties="$properties /p:DotNetBuildWithOnlineSources=true"
```

Comment:
The use of "Sources" here is not descriptive enough to me and is confusing with DotNetBuildSourceOnly.  Thoughts on naming this `DotNetBuildWithOnlineFeeds`?

Reply:
Yeah that's definitely better. Is `DotNetBuildOffline` even better? Or does it imply too much about what the build infra does to restrict internet connections?

Reply:
Too me that implies too much that wouldn't actually happen - e.g. the build doesn't disable the network.

Reply:
Right...alright, BuildWithOnlineFeeds works for me.